server/s3_motr_writer.cc

```diff
@@ -388,15 +388,15 @@ void S3MotrWiter::write_content(
   assert(oid_list.size() == 1);
 
   if (is_object_opened) {
-    write_content();
+    write_content(is_last_write);
   } else {
     open_objects();
```

Comment:
I need to check if we need to pass **is_last_write** flag to open_object() as well (from open_object_successful(), we call write_content())